src/server/repository/assessment/assessment/getCreateSchemaDDL.ts

```diff
@@ -340,3 +344,47 @@ export const getCreateSchemaCycleOriginalDataPointViewDDL = (assessmentCycleSche
           order by odp.country_iso, odp.year
   `
 }
+
+export const getCreateOrReplaceViewCountryUserSummary = (props: { assessment: Assessment; cycle: Cycle }) => {
+  const { assessment, cycle } = props
+  const schemaName = Schemas.getNameCycle(assessment, cycle)
+  return `
+        create or replace view ${schemaName}.country_user_summary as
+        select distinct on (u.uuid, coalesce(ur.country_iso, ui.country_iso))
+            u.uuid,
+            coalesce(ur.country_iso, ui.country_iso)  as country_iso,
+            u.email,
+            u.props->>'name' as name,
+            u.props->>'surname' as surname,
+            concat(
+                    coalesce(u.props->>'name', ''),
+                    case when u.props->>'name' is not null and u.props->>'surname' is not null then ' ' else '' end,
+                    coalesce(u.props->>'surname', '')
+            ) as fullname,
+            u.status,
+            coalesce(to_jsonb(ur.*), '{}'::jsonb)     as role,
+            case
+                when
+                    to_jsonb(ur.*) is null
+                    then
+                    coalesce(to_jsonb(ui.*), '{}'::jsonb)
+                else
+                    '{}'::jsonb
+                end                                     as invitation
```

Comment:
Sounds good, updated

Reply:
`coalesce(to_jsonb(ui.*), '{}'::jsonb)`

This might create empty object, you are right - I didn't see it in the view though
I will fix in upcoming prs